tensorrt_llm/llmapi/utils.py

```
@@ -634,3 +635,54 @@ def amend_api_doc_with_status_tags(cls, method: Callable) -> str:
+        return [v.canonical_value for v in BackendType]
+
+    # Several utils for unified behavior across trtllm-serve and trtllm-bench
+    @staticmethod
+    def print_backend_info(backend: Optional[str]):
+        """ Print the backend info. """
+        backend = backend or BackendType.default_value()
+        logger.info(f"Running with {BackendType(backend)} backend.")
+
+    # TODO[Superjom]: Remove this method after v1.0.0 is released.
+    @staticmethod
+    def get_default_backend_with_warning(backend: Optional[str]) -> str:
```

Ruff (0.12.2)

1 blank line required between summary line and description

(D205)